src/core/etl/src/Flow/ETL/Transformer/RenameEachTransformer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Transformer;
+
+use Flow\ETL\{FlowContext,
+    Row,
+    Rows,
+    Transformer,
+    Transformer\StyleConverter\RenameEntryStrategy
+};
+
+final readonly class RenameEachTransformer implements Transformer
```

Comment:
To keep it consistent with RenameEntryTransfromer this one should be called RenameEachEntryTransformer